crates/ironrdp-cliprdr-native/src/windows/os_clipboard.rs

```diff
@@ -14,16 +14,7 @@ pub(crate) struct OwnedOsClipboard;
 impl OwnedOsClipboard {
     pub(crate) fn new(window: HWND) -> Result<Self, WinCliprdrError> {
         // SAFETY: `window` is valid handle, therefore it is safe to call `OpenClipboard`.
-        if unsafe { OpenClipboard(window) } == FALSE {
-            // Retryable error
-            if get_last_winapi_error() == ERROR_ACCESS_DENIED {
-                return Err(WinCliprdrError::ClipboardAccessDenied);
-            }
-
-            // Unknown critical error
-            return Err(WinCliprdrError::ClipboardOpen);
-        }
-
```

Comment:
We still need this error handling block to report more detailed error cause

Reply:
It's handled by [Error::from_win32()](https://docs.rs/windows-core/latest/windows_core/struct.Error.html#method.from_win32)